docs/section19/19-19.md

```diff
@@ -0,0 +1,94 @@
+# Section 19 - Rule 19-19       
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-19     
+**Rule Description:** For baseline systems 3 through 8, and 11, 12, and 13, the system fan electrical power for supply, return, exhaust, and relief shall be Pfan = bhp × 746/fan motor efficiency. Where, bhp = brake horsepower of baseline fan motor from Table G3.1.2.9; fan motor efficiency = the efficiency from Table G3.9.1 for the next motor size greater than the bhp using a totally enclosed fan cooled motor at 1800 rpm.      
+**Rule Assertion:** Options are Pass/UNDETERMINED/NOT_APPLICABLE     
```

Comment:
Seems like we should also have a FAIL outcome for when the modeled fan power is less than expected without accounting for any pressure drop adjustments.

Reply:
I would like to discuss this. Since we do not know the other pressure drop adjustments I don't think we should fail any projects (unless we can come up with some sort of high limit ceiling, maybe we assume a MERV 15 filter and sound attenuation and energy recovery if indicated in the schema and fail if greater than the fan power assuming these?)

Reply:
No fails, just undetermined and pass outcome per phone call on 1/12/2023

Reply:
@mkarpman Oh, maybe I meant no fails if fan power is greater than what we calculate. I can revise,